File: src/test/java/com/github/xumeng/java/problems/easy/problem0225/MyStack0225Test.java
package com.github.xumeng.java.problems.easy.problem0225;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class MyStack0225Test {

    @Test
    public void test1() {
        MyStack0225 stack = new MyStack0225();
        stack.push(1);
        stack.push(2);
        assertEquals(2, stack.top());
        assertEquals(2, stack.pop());
        assertFalse(stack.empty());
    }

}

File: src/test/kotlin/com/github/xumeng/kotlin/problems/easy/problem0225/MyStack0225Test.kt
package com.github.xumeng.kotlin.problems.easy.problem0225

import org.junit.jupiter.api.Assertions.*
import org.junit.jupiter.api.Test

class MyStack0225Test {

    @Test
    fun test() {
        val stack = MyStack0225()
        stack.push(1)
        stack.push(2)
        assertEquals(2, stack.top())
        assertEquals(2, stack.pop())
        assertFalse { stack.empty() }
    }

}